pkg/windows/windows.go

```diff
@@ -237,7 +242,14 @@ func GetK8sDir() string {
 	return K8sDir
 }
 
-// Windows contains all the methods needed to configure a Windows VM to become a worker node
+// WICDCertConfig contains certificate authentication configuration for WICD
+type WICDCertConfig struct {
```

Comment:
I don't think this needs to be made public. Especially based on how its being used in Nodeconfig.